apex-squid/src/test/java/org/fundacionjala/enforce/sonarqube/apex/parser/grammar/ApexGrammarDmlOperationTest.java

```diff
@@ -23,39 +23,35 @@
  */
 package org.fundacionjala.enforce.sonarqube.apex.parser.grammar;
 
+import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
 import org.junit.Before;
 import org.junit.Test;
-
-import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
-
 import static org.sonar.sslr.tests.Assertions.assertThat;
+import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.DML_OPERATION;
 
-import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.DML_STATEMENT;
-
-public class ApexGrammarDmlStamentTest extends ApexRuleTest {
-
-    @Before
+public class ApexGrammarDmlOperationTest extends ApexRuleTest{
+@Before
     public void init() {
-        setRootRule(DML_STATEMENT);
+        setRootRule(DML_OPERATION);
     }
 
     @Test
-    public void RulesDmlStament() {
+    public void RulesDmlOperation() {
         assertThat(parser)
-                .matches("insert newAcct;")
-                .matches("update myAcct;")
-                .matches("upsert acctList;")
-                .matches("delete doomedAccts;")
-                .matches("undelete savedAccts;");
+                .matches("insert primaryExpression")
```

Comment:
Actually the "semicolon" isn't part of the DML_OPERATION rule, it's part of the STATEMENT rule: 

<p align="left">
    STATEMENT::= <br>
    &nbsp;&nbsp;<a href="block.html">BLOCK</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="dmlOperations.html">DML_OPERATIONS</a>";"<br>
    &nbsp;&nbsp;|&nbsp;<a href="emptyStatement.html">EMPTY_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="compoundStatementExpression.html">COMPOUND_STATEMENT_EXPRESSION</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="ifStatement.html">IF_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="whileStatement.html">WHILE_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="doStatement.html">DO_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="forStatement.html">FOR_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="breakStatement.html">BREAK_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="continueStatement.html">CONTINUE_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="returnStatement.html">RETURN_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="throwStatement.html">THROW_STATEMENT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="tryStatement.html">TRY_STATEMENT</a><br>
</p>